src/Bundle/Renderer/TwigGridRenderer.php

```diff
@@ -113,14 +113,15 @@ public function renderAction(GridViewInterface $gridView, Action $action, $data
     public function renderFilter(GridViewInterface $gridView, Filter $filter)
     {
         $template = $this->getFilterTemplate($filter);
+        $filterName = str_replace('\\', '', $filter->getName());
```

Comment:
Will this form name be visible anywhere, like SF debugger?

Should it be formatted to "some_name" or "someName" instead of current "SomeName"?

Like,
```suggestion
        $filterName = strtolower(str_replace('\\', '_', $filter->getName()));
```
or is there no way to see if it's FQCN instead of regular string?

Reply:
You're right, I think the name will be used for the subname of the criteria. ?criteria[form_name]=value